src/main/java/com/monumental/models/VerificationToken.java

```diff
@@ -0,0 +1,59 @@
+package com.monumental.models;
+
+import com.sun.istack.NotNull;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import java.sql.Timestamp;
+import java.util.Calendar;
+import java.util.Date;
+
+@Entity
+public class VerificationToken extends Model {
+    private static final int EXPIRATION = 60 * 24;
```

Comment:
Is this used anywhere?

Reply:
No, originally they were going to expire but I don't really see much reason to do so